docs/requirements.txt

```diff
@@ -1,7 +1,9 @@
 -r ../requirements.txt
 
 alabaster_jupyterhub
-autodoc-traits
+# Temporary fix of #3021, revert back to released autodoc-traits when
+# possible.
+https://github.com/jupyterhub/autodoc-traits/archive/master.zip
```

Comment:
Maybe use the SHA of the commit on master right now. That way we are "pinned" to a known state. In case someone pushes more things to master in autodoc-traits. What do you think?